docs/en/revocation-lists.rst

```diff
@@ -1,57 +1,520 @@
 .. include:: ../common/common_definitions.rst
 
-.. _revocation-lists.rst:
+.. _sec_revocation_intro:
 
-revocation-lists.rst
-+++++++++++++++++++++++++++
+Credential Lifecycle
+++++++++++++++++++++
 
-[What is it]
-    
-[What it is usefull for]
+The value of a Digital Credential is conditional on its validity. A Credential that has been revoked, due to legal requirements, inaccuracy or compromise, is valueless and potentially harmful. 
+For these reasons a robust mechanism for managing the life-cycle and the revocation of a Digital Credential is required.
+
+This section outlines the key technical requirements and processes related to the revocation of Digital Credentials. 
+Furthermore, it provides the technical details that the Verifiers MUST implement to verify, in a secure and reliable manner, the validity of a Digital Credential during the presentation phase.
+
+The verification of the validity of a Digital Credential is based on the `[OAuth Status Attestation draft 00] <https://datatracker.ietf.org/doc/draft-demarco-oauth-status-attestations/00/>`_ specification. 
+
+A Status Attestation is a signed object that is evidence of the validity status of a Digital Credential. The Issuer provides these attestations to Holders who can present them to Verifiers together with the corresponding Digital Credentials. 
+
+Generally, the Status Attestation enables:
+
+- an entirely automated issuance flow, as the User authentication is not required for the provisioning of the Status Attestation; 
+- the verification of the digital Credential validity status when both the Wallet Instance and the Verifier are offline;
+- better privacy of the User since:
+
+  1. the Verifier cannot check over time the validity of a given  Digital Credential related to the User;
+  2. it prevents issuers to track when and where a Digital Credential is verified.
+  3. it doesn't reveal any information about the Users or the content of their Digital Credentials.
+
+.. _sec_revocation_assumption:
+
+General Assumptions
```

Comment:
```suggestion
Operational Requirements
```
The statements provided are not merely assumptions; they outline operational requirements and roles within a digital credentialing ecosystem